src/main.rs

```diff
@@ -96,42 +95,59 @@ async fn handle_message(
         }
     };
 
-    // StateChange related to Account
-    let account_changes_future = db_adapters::account_changes::store_account_changes(
-        &pool,
-        &streamer_message.state_changes,
-        &streamer_message.block.header.hash,
-        streamer_message.block.header.timestamp,
-    );
-
-    join!(
-        execution_outcomes_future,
-        accounts_future,
-        access_keys_future,
-        account_changes_future,
-    );
+    if strict_mode {
+        // StateChange related to Account
+        let account_changes_future = db_adapters::account_changes::store_account_changes(
+            &pool,
+            &streamer_message.state_changes,
+            &streamer_message.block.header.hash,
+            streamer_message.block.header.timestamp,
+        );
+
+        join!(
+            execution_outcomes_future,
+            accounts_future,
+            access_keys_future,
+            account_changes_future,
+        );
+    } else {
+        join!(
+            execution_outcomes_future,
+            accounts_future,
+            access_keys_future,
```

Comment:
Storing access keys may also fail in non-strict mode, so we should skip them.

P.S. Update the Clap help to indicate this.